src/meltano/core/tracking/environment.py

```diff
@@ -0,0 +1,120 @@
+"""Environment context for the Snowplow tracker."""
+
+from __future__ import annotations
+
+import ctypes
+import ctypes.wintypes
```

Comment:
This seems to not play nice with python 3.7: https://github.com/meltano/meltano/runs/6734175975?check_suite_focus=true , but seems to at least pass tests on 3.8+. Do we need to ensure that this only imports on Windows platforms?

The python docs don't explicitly mention that this should fail on non-windows systems but various posts like https://stackoverflow.com/questions/20897842/error-in-importing-ctypes-wintypes imply just that.

We're currently using this to get a core count on windows - so we could omit it if we really don't actually need the core count anyway.

Reply:
Update. Ended up just nuking it because I wasn't sure how fragile ctypes where and we don't really test on windows (yet). Happy to put it back in (wrapping the import in try/except) though if we want it.

Reply:
@pandemicsyn - Yes, sounds like that is the right call. Sorry I didn't see this earlier.